bootstrap-yum.sh

```diff
@@ -246,15 +252,15 @@ installPostgres() {
 # Install OpenNMS rpm repository for specific release
 installOnmsRepo() {
   echo "📦 Install OpenNMS Repository            ... "
-  curl -1sLf 'https://packages.opennms.com/public/stable/setup.rpm.sh' | sudo -E bash
-  curl -1sLf 'https://packages.opennms.com/public/common/setup.rpm.sh' | sudo -E bash
+  sudo dnf -y install https://yum.opennms.org/repofiles/opennms-repo-stable-rhel9.noarch.rpm
+  sudo sed -i 's/gpgcheck=1/gpgcheck=0/g' /etc/yum.repos.d/opennms-repo-stable-rhel9.repo
 }
 
 ####
 # Install the OpenNMS application from rpm repository
 installOnmsApp() {
   echo -n "📦 Install OpenNMS Horizon packages      ... "
-  sudo dnf -y install rrdtool jrrd2 jicmp jicmp6 opennms-core opennms-webapp-jetty opennms-webapp-hawtio 1>>"${ERROR_LOG}" 2>>${ERROR_LOG}
+  sudo dnf -y install opennms  1>>"${ERROR_LOG}" 2>>${ERROR_LOG}
```

Comment:
Can we keep the individual packages? The opennms meta package is a bit tricky to deal with. The resolution of 3rd party dependencies is sometimes a bit problematic.

Reply:
The original command failed. I didn't dive deeper into it and just followed the installation docs. We just install the `opennms` package there.
But I can give it a try again and check what exactly failed.